### PR TITLE
refactor: rename NEXT_TABLE_ROUTE_PREFIX to TABLE_ROUTE_PREFIX

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

1. rename NEXT_TABLE_ROUTE_PREFIX to TABLE_ROUTE_PREFIX
2. fix parsing `full_table_name` incorrectly

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: src/meta-srv/src/service/admin/route.rs [hunk 1]
```diff
@@ -38,51 +35,44 @@ impl HttpHandler for RouteHandler {
...
+                .context(error::MissingRequiredParameterSnafu {
+                    param: "full_table_name",
+                })?;
+
+        let (catalog, schema, table) =
+            parse_full_table_name(table_name).context(error::InvalidFullTableNameSnafu)?;
+
+        let key = TableNameKey::new(catalog, schema, table);
+
+        let mut result = HashMap::with_capacity(1);
```
Comment (fengjiachun) [inline src/meta-srv/src/service/admin/route.rs:50]: Better to return `TableRouteValue` directly

**Review by fengjiachun — APPROVED** [review]

**Review by fengjiachun — APPROVED** [review]
LGTM

**Comment by WenyXu** [comment]
@fengys1996 @shuiyisong @MichaelScofield PTAL

**Review by shuiyisong — APPROVED** [review]
LGTM

**Pull request merged**